packages/stacked_tools/lib/src/commands/create/create_app_command.dart

```diff
@@ -20,21 +25,34 @@ class CreateAppCommand extends Command {
   @override
   String get name => 'app';
 
+  CreateAppCommand() {
+    argParser.addFlag(
+      ksV1,
+      aliases: [ksUseBuilder],
+      defaultsTo: null,
+      help: kCommandHelpV1,
+    );
+  }
+
   @override
   Future<void> run() async {
-    await locator<ConfigService>().loadConfig();
+    await _configService.loadConfig();
 
     final appName = argResults!.rest.first;
 
     await _processService.runCreateApp(appName: appName);
 
+    /// Removes `widget_test` file to avoid failing unit tests on created app
+    await _fileService.deleteFile(filePath: '$appName/test/widget_test.dart');
```

Comment:
Interesting. 

We should add this to the template actions and add a delete action. This way we can provide a "step" in the template that deletes a file as well. 

This solution works for now. If we need to do another deletion we can change it into a delete action in the template, which I think we have already.